vcd/sample_vcd_test_config.json

```diff
@@ -90,6 +90,45 @@
     "uploadPieceSize": 5,
     "uploadProgress": true
   },
+  "testEnvBuild": {
+    "//": "This section is optional for running tests. It's only needed to build the test environment in a new vCD",
+
+    "// ": "Entities that will be created in an empty vCD use names defined above",
+
+    "// ": "org:              from vcd.org",
+    "// ": "vdc:              from vcd.vdc",
+    "// ": "catalog:          from vcd.catalog.name",
+    "// ": "catalogItem:      from vcd.catalog.catalogItem",
```

Comment:
"catalogItem" -> "catalog item" (I believe there was no intention to make this look like a specific variable).

Reply:
fixed